From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: deepak1556 <hop2deep@gmail.com>
Date: Fri, 30 Apr 2021 16:36:14 -0700
Subject: Revert "Roll clang llvmorg-13-init-7051-gdad5caa5-2 :
 llvmorg-13-init-7296-ga749bd76-1"

This reverts commit 6da81f26271d1efe8c3da6dd7cfb409d35763a35.

This broke WoA builds, ref: https://github.com/electron/electron/issues/28876

diff --git a/tools/clang/scripts/update.py b/tools/clang/scripts/update.py
index a4ae8fa31330c92a356d8fd9e67fc49826a2b5d6..33cd83fc835e0204a4aba0e17d2152b3021a4c35 100755
--- a/tools/clang/scripts/update.py
+++ b/tools/clang/scripts/update.py
@@ -39,8 +39,8 @@ import zipfile
 # https://chromium.googlesource.com/chromium/src/+/main/docs/updating_clang.md
 # Reverting problematic clang rolls is safe, though.
 # This is the output of `git describe` and is usable as a commit-ish.
-CLANG_REVISION = 'llvmorg-13-init-7296-ga749bd76'
-CLANG_SUB_REVISION = 3
+CLANG_REVISION = 'llvmorg-13-init-7051-gdad5caa5'
+CLANG_SUB_REVISION = 2
 
 PACKAGE_VERSION = '%s-%s' % (CLANG_REVISION, CLANG_SUB_REVISION)
 RELEASE_VERSION = '13.0.0'
